Demo/Demo/FeatureViewControllers/SwiftUICardDemoView.swift

```diff
@@ -0,0 +1,114 @@
+import SwiftUI
+import PaymentsCore
+import Card
+
+enum Fields {
+    case cardNumber
+    case expirationDate
+    case cvv
+}
+
+@available(iOS 13.0.0, *)
+struct SwiftUICardDemo: View {
+
+    // MARK: Variables
+
+    @State private var cardNumberText: String = ""
+    @State private var expirationDateText: String = ""
+    @State private var cvvText: String = ""
+
+    @StateObject var baseViewModel = BaseViewModel()
+
+    private let cardFormatter = CardFormatter()
+
+    // MARK: Views
+
+    var body: some View {
+        ZStack {
+            FeatureBaseViewControllerRepresentable(baseViewModel: baseViewModel)
+            VStack(spacing: 16) {
+                FloatingLabelTextField(placeholder: "Card Number", text: $cardNumberText)
+                    .onChange(of: cardNumberText) { newValue in
+                        formatFieldWith(newValue, field: .cardNumber)
+                    }
+                FloatingLabelTextField(placeholder: "Expiration Date", text: $expirationDateText)
+                    .onChange(of: expirationDateText) { newValue in
+                        formatFieldWith(newValue, field: .expirationDate)
+                    }
+                FloatingLabelTextField(placeholder: "CVV", text: $cvvText)
+                    .onChange(of: cvvText) { newValue in
+                        formatFieldWith(newValue, field: .cvv)
+                    }
+                Button("\(DemoSettings.intent.rawValue.capitalized) Order") {
+                    guard let card = createCard(), let orderID = baseViewModel.orderID else {
+                        baseViewModel.updateTitle("Failed: missing card / orderID.")
+                        return
+                    }
+
+                    checkoutWithCard(card, orderID: orderID)
+                }
+                .foregroundColor(.white)
+                .padding()
+                .frame(maxWidth: .infinity)
+                .background(.blue)
+                .cornerRadius(10)
+                .disabled(!(cvvText.count >= 3 && cardNumberText.count >= 17 && expirationDateText.count == 7))
+            }
+            .padding(.horizontal, 16)
+        }
+    }
+
+    // MARK: Functions
+
+    private func formatFieldWith(_ text: String, field: Fields) {
```

Comment:
Can we put button enable/disable logic here to make behavior consistent with uikit card demo?

Reply:
So we _can_ but this isn't best practice in SwiftUI. SwiftUI uses the `disabled` attribute on the button, which I have implemented here:
```
.disabled(!(cvvText.count >= 3 && cardNumberText.count >= 17 && expirationDateText.count == 7))
```
We can do either but I went with this because it seems to be the preferred method with Swift UI. The button is disabled in the same way as the other card demo, but just in each UI Frameworks preferred way.

Reply:
ahhh nice, that makes sense.
Maybe we can add the check for `orderID != nil` here?

and I wonder if we can grey out the text when it's disabled, like `.foregroundColor(<bool to see if button should be enabled> ? .white : .gray)` or something similar?

Reply:
Updated and made some updates to the enable pay button function so it can be shared across the two frameworks